### PR TITLE
#41: fix order tables formatting

### PR DESCRIPTION
**Pull request by tobyash86**

### CODE REVIEW

File: WebGoatCore/Views/Checkout/Receipt.cshtml [hunk 1]
```diff
@@ -10,51 +10,51 @@
...
-                        <th>Product Name</th>
-                        <th>Unit Price</th>
-                        <th>Quantity</th>
-                        <th>Extended Price</th>
+                    <tr class="headerRow">
+                        <th class="tabHeader">Product Id</th>
+                        <th class="tabHeader">Product Name</th>
+                        <th class="tabHeader">Unit Price</th>
+                        <th class="tabHeader">Quantity</th>
+                        <th class="tabHeader">Extended Price</th>
```
Comment (pkruk2) [inline WebGoatCore/Views/Checkout/Receipt.cshtml:20]: I think there is no need to add class to this th elements. The headerRow already defines the row as the header, so you should be able to apply CSS with something like `tr.headerRow th` (https://www.w3schools.com/css/css_combinators.asp)

File: WebGoatCore/Views/Checkout/Receipt.cshtml [hunk 2]
```diff
@@ -10,51 +10,51 @@
...
                                 <a asp-controller="Product" asp-action="Details" asp-route-productId=@orderDetails.ProductId>@orderDetails.ProductId</a>
                             </td>
-                            <td>@orderDetails.Product.ProductName</td>
-                            <td>@orderDetails.DecimalUnitPrice</td>
-                            <td>@orderDetails.Quantity</td>
-                            <td>@orderDetails.ExtendedPrice</td>
+                            <td class="tabCell">@orderDetails.Product.ProductName</td>
+                            <td class="tabCell">@orderDetails.DecimalUnitPrice.ToString("F")</td>
+                            <td class="tabCell">@orderDetails.Quantity</td>
+                            <td class="tabCell">@orderDetails.ExtendedPrice.ToString("F")</td>
```
Comment (pkruk2) [inline WebGoatCore/Views/Checkout/Receipt.cshtml:31]: Similar here. "td" already means "table cell". 
And in other places.

File: WebGoatCore/Views/Shared/_CartPartial.cshtml [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
-                <td>@order.DecimalUnitPrice</td>
-                <td>
+                <td class="tabCell">@order.ProductId</td>
+                <td class="tabCell">@order.Product.ProductName</td>
+                <td class="tabCell">@order.DecimalUnitPrice.ToString("F")</td>
+                <td class="tabCell">
                     <input type="text" value=@order.Quantity class="quantityInput">
                 </td>
-                <td class="extendedPrice">@order.ExtendedPrice</td>
+                <td class="tabCell">@order.ExtendedPrice.ToString("F")</td>
```
Comment (pkruk2) [inline WebGoatCore/Views/Shared/_CartPartial.cshtml:21]: This looks suspicious. Previously the class defined this cell as special cell (with extended price). I suppose the intention was to format it in a special way (maybe in bold, maybe with larger font). This change makes it a normal cell. (Plus, as mentioned in other comment, "td" is already "table cell")

**Review by pkruk2 — CHANGES_REQUESTED** [review]

**Comment by tobyash86** [comment]
Thanks, I have fixed stylesheet usage. Please review again.

File: WebGoatCore/Views/Checkout/Receipt.cshtml (hunk 2 above)
Comment (tobyash86) [reply WebGoatCore/Views/Checkout/Receipt.cshtml]: done

File: WebGoatCore/Views/Shared/_CartPartial.cshtml (hunk 3 above)
Comment (tobyash86) [reply WebGoatCore/Views/Shared/_CartPartial.cshtml]: there was no such style defined in css / done

**Review by pkruk2 — APPROVED** [review]
Look ok now

**Pull request merged**